src/app/routes/applications/new-application/new-application.component.html

```diff
@@ -40,16 +40,18 @@ <h4 mat-dialog-title class="mb-0 d-flex justify-content-between align-items-cent
         <ng-container *ngFor="let step of txs; let i = index">
           <mat-step [editable]="false" [completed]="false">
             <ng-template matStepLabel>{{ step.info }}</ng-template>
-            <app-card-info [description]="step.info"></app-card-info>
-            <div
-              class="card card-hint d-flex flex-row justify-content-start align-items-center shadow-none d-flex pt-3 pb-2 mb-3">
-              <mat-icon class="px-2">notifications_none</mat-icon>
-              <span class="pl-3 pb-2"> Please confirm this transaction in your connected Wallet. </span>
-            </div>
-            <div class="d-flex justify-content-end mb-3">
-              <button mat-stroked-button class="btn btn-small mr-2" (click)="closeDialog()">Cancel</button>
-              <button mat-raised-button class="btn btn-small btn-primary" (appRetryBtn)="retry()">Retry</button>
-            </div>
+            <ng-container *ngIf="stepper?.selectedIndex === i + 2">
```

Comment:
@dawidgil I don't quite get it. Why the `2` here?

Reply:
We have 2 steps before ^^ First for describing values for application, second for summary. That's why there is 2. It's far from ideal, but i do not want to refactor to much, so i'm doing my best to adjust it.